README.md

```diff
@@ -5,39 +5,391 @@
+# (Equivalent)
+#   COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile-arm" docker compose -p issuer -f /Users/username/path/to/sh-id-platform/infrastructure/local/docker-compose.yml up -d initializer;
+#	sleep 5;
+#   DID=$(docker logs -f --tail 1 issuer-initializer-1 | grep "did");
+#   echo $DID;
+#   sed '/ISSUER_API_UI_ISSUER_DID/d' .env-api > .env-api.tmp;
+#   echo ISSUER_API_UI_ISSUER_DID=$DID >> .env-api.tmp;
+#   mv .env-api.tmp .env-api;
+#   docker rm issuer-initializer-1;
+
+# Expected Output:
+#   COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile-arm" docker compose -p issuer -f /Users/username/path/to/sh-id-platform/infrastructure/local/docker-compose.yml up -d initializer
+#   WARN[0000] Found orphan containers ([issuer-vault-1 issuer-postgres-1 issuer-redis-1]) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up. 
+#   [+] Running 1/1
+#   ⠿ Container issuer-initializer-1  Started                                                                                0.2s
+#   sleep 5
+#   did:polygonid:polygon:mumbai:uniqueAlphanumericKeyGenerated
+#   sed '/ISSUER_API_UI_ISSUER_DID/d' .env-api > .env-api.tmp
+#   mv .env-api.tmp .env-api
+#   docker rm issuer-initializer-1
+#   issuer-initializer-1
+```
+
+#### Start Issuer Node API
+
+Now that you have all your files configured we can now start the issuer node api itself.
+
+> For _NON-Apple-M1/M2/Arm_ (ex: Intel/AMD):
+
+```bash
+# FROM: ./
+
+make run;
+# (Equivalent)
+#   COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile" docker compose -p issuer -f /Users/username/path/to/sh-id-platform/infrastructure/local/docker-compose.yml up -d api;
+
+# Expected Output:
+#   COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile" docker compose -p issuer -f /Users/username/path/to/sh-id-platform/local/docker-compose.yml up -d api;
+```
+
+> For _Apple-M1/M2/Arm_:
+
+```bash
+# FROM: ./
+
+make run-arm;
+# (Equivalent)
+#   COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile-arm" docker compose -p issuer -f /Users/username/path/to/sh-id-platform/infrastructure/local/docker-compose.yml up -d api;
+
+# Expected Output:
+#   COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile-arm" docker compose -p issuer -f /Users/username/path/to/sh-id-platform/local/docker-compose.yml up -d api;
+#   WARN[0000] Found orphan containers ([issuer-vault-1 issuer-postgres-1 issuer-redis-1]) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up. 
+```
+
+Now open up [http://localhost:3001](http://localhost:3001) and see the following:
+
+![Issuer Node API](/docs/3001.png)
+
+#### Configure UI Environment Variables
+
+This is to configure our UI admin to manage our credentials as an issuer at [http://localhost:8088](http://localhost:8088)
+
+> **NOTE:** REQUIRED STEP in order for the ui to work properly.
```

Comment:
This is referencing the wrong file. ` ./ui/.env` is used for [development](https://github.com/0xPolygonID/sh-id-platform/pull/327/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R604) only. For live (:8088), the instructions used to be in "Configuration" [section](https://github.com/0xPolygonID/sh-id-platform/pull/327/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R604), but you removed them.

```
`ISSUER_UI_BLOCK_EXPLORER_URL`
`ISSUER_UI_AUTH_USERNAME`
`ISSUER_UI_AUTH_PASSWORD`
```